niceml/data/datasets/dfdataset.py

```diff
@@ -100,19 +220,34 @@ def get_all_data(self):
     def extract_data(self, cur_indexes, cur_input):
         """extracts data"""
```

Comment:
Please add more docstring to give an idea about what data is extracted.

Reply:
I've rewritten the docstring